documentation/src/main/markdown/runningdetect/threatintelscan.md

```diff
@@ -0,0 +1,45 @@
+# [threat_intel] Scan
+
+[threat_intel] Scans are a way of running binary file analysis that provides malware warnings, with a risk analysis level applied, for open source and commercial software.
+
+[company_name] [solution_name] will accept a user provided local file path to a binary file for [threat_intel] Scan. This file may be a single executable, or a compressed file, such as a tar or zip, that contains many files for analysis.
+
+Identification of malware will be displayed to [blackduck_product_name] users will include file name, file path, and other identifiers, along with a description of the type of malware, and severity of the finding if malware is detected.
+
+## Workflow
+
+1. [company_name] [solution_name] sends a BDIO header to scan service to generate a scan ID.   
+1. The file for scanning is uploaded to [blackduck_product_name] Storage service.   
+1. Once uploaded, [threat_intel] service container receives a RabbitMQ message, indicating that there is a new scan request, at which point [threat_intel] service takes the file from Storage service and downloads it to its own container.   
```

Comment:
TMI about RabbitMQ message. Would it not be enough if we describe as "Once uploaded, [threat_intel] service container downloads a copy of the file from Storage service."

Reply:
I agree with @gopannabd  there may be too much detail in this section. Below is what I suggest, but feel free to tweak as you see fit Chris:
1. Internal detail not relevant to customer
2. [👍](https://emojipedia.org/thumbs-up)
3. Madhu's suggestion sounds good to me *IF* that is the language/level of detail we use elsewhere (specifically mentioning microservices) 
4. [threat intel] Service invokes ReversingLabs tools to analyze the hash of this file for malware and threat intelligence. 
5. Minor: I see we say "Black Duck Scan Service" and "Black Duck Storage Service" ... Should we adopt the same for "Threat Intel Service"? 


(Also I see above the steps are all numbers "1." -- is that a typo? Not sure if it'll be displayed differently in the actual docs)

Minor: The sentence at line 7 needs rewording, maybe "Identification of malware ~will be~ displayed to [BD] users will include file name..."